collector/mongodb_collector.go

```diff
@@ -9,9 +11,17 @@ import (
 
 var (
 	// Namespace is the namespace of the metrics
-	Namespace = "mongodb"
+	Namespace = namespace()
 )
 
+func namespace() string {
+	n := os.Getenv("NAMESPACE")
```

Comment:
please make this name more specific, something like `MONGODB_EXPORTER_NAMESPACE`